ciw/node.py

```diff
@@ -125,6 +128,9 @@ def begin_service_if_possible_accept(self, next_individual):
             - Update the server's end date (only when servers are not infinite)
         """
         next_individual.arrival_date = self.get_now()
+        if self.reneging:
```

Comment:
```suggestion
        if self.reneging is True:
```

Reply:
Re that: https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html